x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/event_merging_banner.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCallOut, EuiLink, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+
+export const EventMergingBanner: React.FC<{ onDismiss: () => void }> = ({ onDismiss }) => {
```

Comment:
please memoize this component with the use of `memo()`

Reply:
Also - can you add a unit test file for this component

Reply:
memoized: 78d41f1c7e8982ca790f3b91fbdadfcb75be60b0
unit test: 80c7cfcd1bf4e09c6a6d58883badc6b6d22b9069